### DISCUSSION TITLE
Correlation Matrix Plot

### DISCUSSION THREAD

**Discussion by monafaraji** [MAIF/shapash#187] [Q&A]
Hi, 
in the example report for the tutorial there is a Correlation matrix plot for House price data, would you please help me find the method for it?
it seems it's missing in Documentation >> THe plot methods here
https://shapash.readthedocs.io/en/latest/autodocs/shapash.explainer.html#the-plot-methods

Many thanks

**Comment by ThibaudReal** [comment]
Hello monafaraji,

Thank you for your comment. 
The correlation matrix plot in the House Price example was generated using a function that is only used in the Shapash Report. 
Unfortunately this function is not available as a method of the SmartPlotter object for now but **we are planning on adding it soon.**
I will write the corresponding issue to add this feature.
